### DISCUSSION TITLE
stitches migration; the `as` prop.

### DISCUSSION THREAD

**Discussion by hainessss** [chakra-ui/panda#2754] [Q&A] [answered]
Hello, I am migrating our stitchesJS styled component library to Panda CSS. the `styled` function in stitches returns a component with an `as` prop so that you could do things like..

```
<Box as='img' />
or
<Box as='span' />
```

it doesn't look like the component returned by the panda `styled` function has that prop? I am wondering what would be the best way to convert these components to Panda. Thanks!

**Comment by anubra266** [comment, marked as answer]
@hainessss

Check out this playground:
https://play.panda-css.com/JzPVnnizuC

```jsx
const Img = styled('img')
const span = styled('span')
```

 You might find this doc helpful
https://panda-css.com/docs/migration/stitches#styled-function

**Reply by hainessss** [reply]
okay so the answer is to just create helpers for each element

**Reply by omarkhatibgg** [reply]
better option is to use `styled.span` or `styled.img`.

for button or things that needs custom style use radix asChild component
